airbyte-proxy/README.md

```diff
@@ -0,0 +1,7 @@
+# Airbyte Proxy
+
+This service uses Nginx to front the Aribyte `webapp` and `server` services to add Authentication via HTTP basic auth.
```

Comment:
Can we add more detail here about how this works?

Two points I'm thinking of:
- intended to work in conjunction with the `airbyte_internal` network defined in the default docker compose file.
- by default forwards requesting coming in on 8000 and 8001 to the `PROXY_PASS_WEB` and `PROXY_PASS_API` accordingly.

Otherwise it's two hops to understand how this fits in with the general docker deploy.